src/components/AddBlockButton/AddBlockButton.tsx

```diff
@@ -39,12 +49,16 @@ export const AddBlockButton = ({
             className={merge([
                 'tw-group tw-leading-none tw-rounded-sm tw-outline-none',
                 isFocusVisible && FOCUS_STYLE,
-                orientation === AddBlockButtonDirection.Vertical ? 'tw-rotate-90' : '',
+                orientation === AddBlockButtonDirection.Vertical ? 'tw-rotate-90' : 'tw-rotate-0',
```

Comment:
Here I would just add maybe `!direction && orientation ...`
So if direction is defined, orientation gets overwritten.

Reply:
Done